lib/new_relic/agent/transaction/trace_context.rb

```diff
@@ -132,11 +132,17 @@ def accept_trace_context(header_data)
           transaction.trace_id = header_data.trace_id
           transaction.parent_span_id = header_data.parent_id
 
-          return false unless payload = assign_trace_state_payload
+          trace_flags = header_data.trace_parent['trace_flags']
+          payload = assign_trace_state_payload
 
-          transaction.distributed_tracer.parent_transaction_id = payload.transaction_id
+          if payload
+            determine_sampling_decision(payload, trace_flags)
+          else
+            determine_sampling_decision(TraceContextPayload::INVALID, trace_flags)
```

Comment:
Is this understanding true - the constant `TraceContextPayload::INVALID` is a new instance of TraceContextPayload, which is the result of the `create` method? I think the name `INVALID` is tripping me up.